edx_solutions_api_integration/users/serializers.py

```diff
@@ -178,6 +178,12 @@ class CourseProgressSerializer(serializers.Serializer):
     def get_progress(self, enrollment):
         completion_percentage = 0
 
+        # If this enrollment has a value from the new API, use that instead of calculating one.
+        if enrollment['course_id'] in self.context['new_api_data']:
+            return self.context['new_api_data'][enrollment['course_id']]['completion']['ratio']
```

Comment:
The old API returns the completion value as a percentage, but the new API returns a ratio. Shouldn't we multiply the ratio by 100 here to make sure the old API still returns a percentage?

Reply:
I agree. I was confused why we would change the output type.

Reply:
Did I suggest that?  I must have been confused.  I agree that this should continue to return a value between 0 and 100.

Reply:
Its no problem Cliff. There was quite a bit of confusion with this one overall. I should be getting a fix up here soon, getting ready to build a few tests.